README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
```

Comment:
This should be as "This repository contains following resources :" to be more perfect

Reply:
+1